app-text/wiki/wiki-1.4.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GOLANG_PKG_IMPORTPATH="github.com/walle"
+
+GOLANG_PKG_DEPENDENCIES=(
+	"github.com/mattn/go-colorable:6fcc0c1"
+	"github.com/mattn/go-isatty:6ca4dbf"
+	)
+
+inherit golang-single
+
+DESCRIPTION="Command line tool to fetch summaries from MediaWiki"
+HOMEPAGE="https://github.com/walle/wiki"
```

Comment:
There is no need to declare the `HOMEPAGE` when it is equal to to the `GOLANG_PKG_IMPORTPATH` + $PN; it will be added automatically by the eclass ;)